docs/lang/articles/get-started/index.md

```diff
@@ -3,330 +3,198 @@ sidebar_position: 1
 slug: /
 ---
 
-# Getting Started
-
-## Target audience
-
-End users who *only* wish to quickly set up Taichi for simulation or high-performance numerical computation.
-
-:::caution IMPORTANT
-
-For developers who are interested in the compiler, computer graphics, or high-performance computing, and would like to contribute new features or bug fixes to the [Taichi programming language](https://github.com/taichi-dev/taichi), see the [Developer installation](../contribution/dev_install.md) for more information on building Taichi from source.
-
-:::
-
-## Prerequisites
-
-### Python
-
-3.7/3.8/3.9/3.10 (64-bit)
-
-:::note
 
-Taichi recommends installing Python from [Miniforge](https://github.com/conda-forge/miniforge/#download) conda if you are on a MacBook with M1 chip.
-
-:::
+# Getting Started
 
-### Supported systems and backends
+Taichi is a high-performance parallel programming language using Python as the frontend. The commom usage is, users write their computation-intensive tasks in python obeying a few extra rules imposed by Taichi and use the two decorators `ti.func` and `ti.kernel` to tell Taichi to take over the functions that implement the task. Taichi's just-in-time compiler will compile these functions to machine code and all subsequent calls to them are executed on multi-CPU cores or GPU. In typical scenarios like physical simulations or real-time renderings, this will usually give a 50~100x speed up compared to native python!
```

Comment:
```suggestion
Taichi is a high-performance parallel programming language using Python as the frontend. Taichi users write their computation-intensive tasks in Python obeying a few extra rules imposed by Taichi, and use the two decorators `ti.func` and `ti.kernel` to tell Taichi to take over the functions that implement the task. Taichi's just-in-time (JIT) compiler will compile these functions to machine code and all subsequent calls to them are executed on multi-CPU cores or GPUs. In typical compute-intense scenarios (such as numerical simulations), this will usually give a 50-100x speed up over native python!
```